leakcanary-object-watcher-android/src/main/java/leakcanary/AppWatcher.kt

```diff
@@ -81,8 +81,11 @@ object AppWatcher {
           changedFields += "${field.name}=$newValue"
         }
       }
-      "Updated AppWatcher.config: Config(${if (changedFields.isNotEmpty())
-        changedFields.joinToString(", ") else "no changes"})"
+
+      val changesInConfig =
+        if (changedFields.isNotEmpty()) changedFields.joinToString(", ") else "no changes"
```

Comment:
What is this change about?

Reply:
Multi-line `if-else` was reported by KtLint as missing braces; I extracted statement into a variable, this allowed to keep it single-line and therefore no need for braces